ksqldb-functional-tests/src/test/resources/query-validation-tests/formats.json

```diff
@@ -291,11 +316,14 @@
       }
     },
     {
-      "name": "override formats - create stream as select",
+      "name": "override formats - create stream as select (old planner)",
       "statements": [
         "CREATE STREAM INPUT (K STRING KEY, foo INT) WITH (kafka_topic='input_topic', key_format='KAFKA', value_format='AVRO');",
         "CREATE STREAM OUTPUT WITH (format='JSON') AS SELECT * FROM INPUT;"
       ],
+      "properties": {
+        "ksql.new.query.planner.enabled": "false"
```

Comment:
We added this config to immutable properties what actually disallows to set for QTT. Not sure if there is some workaround? QTT should be some kind of super-user?